### DISCUSSION TITLE
My Kubernetes Nodes are going to "NoSchedule" and my environment is getting down frequently

### DISCUSSION THREAD

**Discussion by Manikanta-Reddy-Pasala** [mysticaltech/terraform-hcloud-kube-hetzner#361] [Q&A] [answered]
Hello Everyone,

I am new to Terraform. I started using this Kube hetzner to create QA and production for my company.
But my nodes are not distributing the loads properly. Sometimes one node will be left with no containers and the other 2 nodes take full container load and then go into "NoSchedule" mode making my production down. 

Even today it was down, I had to quickly login to hetzner console and restart the machine then within 2 minutes everything comes to normal.

I have deleted my old cluster since I started with the terraform scripts. the cluster was stable for 1 week in QA then I migrated to production as well. now I am kind of blocked. 

I have attached my terraform script below for reference also a screenshot of nodes with unequal load distribution.

<img width="868" alt="Screenshot 2022-10-22 at 9 15 55 AM" src="https://user-images.githubusercontent.com/56497851/197326055-2b797eab-3665-4d08-8dfa-fbb97df1fbad.png">
[kube.txt](https://github.com/kube-hetzner/terraform-hcloud-kube-hetzner/files/9843785/kube.txt)

**Comment by mysticaltech** [comment, marked as answer]
@Manikanta-Reddy-Pasala Just remove the NoSchedule taints from your nodepool definition, this was wrongly in the `kube.tf.example` when you installed it, but now we have removed it as it should not have been there in the first place!
